src/gui/explorer.go

```diff
@@ -182,15 +182,9 @@ func getTransactionsForAddress(gateway Gatewayer) http.HandlerFunc {
 					return
 				}
 
-				uxout, err := gateway.GetUxOutByID(id)
```

Comment:
This function creates the response of `/explorer/address` and already included the inputs data. It was modified to use the functions that were added in gateway.go and thus comply with DRY.